Swift.sublime-syntax

```diff
@@ -344,10 +344,10 @@ contexts:
       scope: keyword.operator
     - match: \w+
       scope: support.class
-    - match: '>'
+    - match: '>\s*'
       pop: true
   class_type:
-    - match: \w+
+    - match: \w+\s*
```

Comment:
Strange indeed!

I'm using the default theme (`Monokai (SL).tmTheme`), which _may_ have some more error highlighting? Could you revert back to Monokai to see if it makes a difference?